Base/home/anon/.config/SystemServer.ini

```diff
@@ -1,3 +1,9 @@
+[LaunchServer]
+Socket=/tmp/100/portal/launch
```

Comment:
I'm assuming that `anon`'s user id is 100? maybe a future PR should add a syntax for `%uid` or similar. it feels weird to hardcode that 100 in the code of all the Applications.

Reply:
You're right!
I absolutely agree with you, and this is what meant by 
> remove the hard-coded portal directory path